documentation/tutorials/1-4-enable-tls.md

```diff
@@ -0,0 +1,79 @@
+## Transport Layer Security (TLS)
+
+[TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) is used to encrypt data exchanged between two applications; it secures data transmitted over the network. Typically, enabling TLS within a highly available database, and between a highly available database and client/server applications, requires domain-specific knowledge and a high level of expertise. Fortunately, the domain-specific knowledge has been encoded into Charmed OpenSearch. This means enabling TLS on Charmed Opensearch is easily available and requires minimal effort on your end.
+
+TLS is enabled via relations, by relating Charmed OpenSearch to the [TLS Certificates Charm](https://charmhub.io/tls-certificates-operator). The TLS Certificates Charm centralises TLS certificate management in a consistent manner and handles providing, requesting, and renewing TLS certificates.
+
+### Configure TLS
+
+Before enabling TLS on Charmed OpenSearch we must first deploy the `tls-certificates-operator` charm:
+
+```bash
+juju deploy tls-certificates-operator
+```
+
+Wait until the `tls-certificates-operator` is ready to be configured. When it is ready to be configured `watch -c juju status --color` will show:
```

Comment:
On Mehdi's recommendation I swapped these commands to use `watch -c juju status` because it has a nicer UI, since it doesn't flicker on update (in this case, once every second). Is there a reason why the built-in `--watch` flag is preferred?